automerge/src/decoding.rs

```diff
@@ -52,7 +52,60 @@ pub enum Error {
     Io(#[from] io::Error),
 }
 
-#[derive(thiserror::Error, Debug)]
+impl PartialEq<Error> for Error {
```

Comment:
What's the reason for having to write this out by hand?

Reply:
The leb and io errors don't implement PartialEq unfortunately